source/administration/command-line-tools.rst

```diff
@@ -0,0 +1,875 @@
+Command Line Tools
+==================
+
+From the directory where the Mattermost platform is installed, a
+``platform`` command is available for configuring the system, including:
+
+**General Administration**
+
+-  Creating teams
+-  Creating users
+-  Assigning roles to users
+-  Reseting user passwords
+-  Inviting users to teams
+
+**Advanced Administration**
+
+-  Permanently deleting users (use cautiously - database backup
+   recommended before use)
+-  Permanently deleting teams (use cautiously - database backup
+   recommended before use)
+
+**Advanced Automation**
+
+*Available in Enterprise Edition E10 and higher*
+
+-  Creating channels
+-  Inviting users to channels
+-  Removing users from channels
+-  Listing all public channels and private groups for a team
+-  Restoring previously deleted channels
+-  Migrating sign-in options
+-  Resetting multi-factor authentication for a user
+
+CLI Commands
+------------
+
+Typing ``platform gendocs`` returns documentation for the CLI tool.
```

Comment:
Maybe instead include a note about `platform help` instead?  You can type `platform help [command]` to get help on any command.